CMakeLists.txt

```diff
@@ -1123,7 +1125,7 @@ if(APPLE AND EXISTS "/usr/bin/otool")
         target_compile_definitions(libfastfetch PUBLIC FF_LIBSYSTEM_VERSION="${CMAKE_MATCH_1}")
     endif()
 endif()
-if(FreeBSD AND EXISTS "/usr/local/bin/objdump")
+if((FreeBSD OR MidnightBSD) AND EXISTS "/usr/local/bin/objdump")
     execute_process(COMMAND /bin/sh -c "/usr/local/bin/objdump -T /lib/libc.so.* | grep 'FBSD_[0-9][0-9]*\\.[0-9][0-9]*' -o | sort -Vru | head -1"
```

Comment:
Does MidnightBSD use FreeBSD libc?

Reply:
MidnightBSD uses FreeBSD libc with some additions.  So there are FBSD symbol versions and also MBSD versions.

Reply:
I guess using MBSD versions is better.